src/nvim/CMakeLists.txt

```diff
@@ -183,7 +189,7 @@ foreach(sfile ${NEOVIM_SOURCES}
     DEPENDS "${HEADER_GENERATOR}" "${sfile}"
     )
   list(APPEND NEOVIM_GENERATED_SOURCES "${gf1}")
-  list(APPEND NEOVIM_GENERATED_SOURCES "${gf2}")
```

Comment:
@ZyX-I I have second thought that NEOVIM_GENERATED_HEADERS should not be appended to NEOVIM_GENERATED_SOURCES. That is, when NEOVIM_GENERATED_SOURCES is used to declare dependencies, NEOVIM_GENERATED_HEADERS should be used there explicitly.
It should make the source easier to be understood at sight.